config/rbac/role.yaml

```diff
@@ -1,4 +1,136 @@
 
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  creationTimestamp: null
+  name: manager-role
```

Comment:
Given that this is a ClusterRole, you might want to rename this to something more descriptive, such as `gatekeeper-manager` or something of the sort. This way it's easier to diferentiate from other clusterRoles

Reply:
Yep, that's how it works. This is generated code by `controller-gen` via `make manifests` and is then installed by kustomize using `make deploy`, which prefixes all resource names with `gatekeeper-operator-`. See below for where this is configured:
https://github.com/font/gatekeeper-operator/blob/88331f30794dd6e88cd7a710c8668aec592a54e0/config/default/kustomization.yaml#L4-L9